new-packaging/src/debian/skeleton/debian/neo4j.default

```diff
@@ -7,10 +7,11 @@
 #
 #     systemctl edit neo4j.service
 #
+# This is automatically done upon upgrade.
```

Comment:
I think this would be more helpful if it was a bit more verbose. Something like:

```
# Any values that were present in this file have been automatically migrated into
# SystemD configuration as part of the upgrade.
```